### ISSUE TITLE
Show number of package imports in status bar 

### ISSUE THREAD

**Issue by dastergon** [Microsoft/vscode-go#1819]
At the bottom of a every godoc.org page, there's an autogenerated sentence that mentions some package stats regarding the number of imported packages and imports by other packages. For [example](https://godoc.org/github.com/hashicorp/vault/api):
```
Package api imports 33 packages (graph) and is imported by 370 packages. 
```
It would be great to support [godepgrap](https://github.com/kisielk/godepgraph), or some other tool, to get the number of imports of that package, and show that number at the editor's status bar (or some other meaningful place). 

An example sentence could look like: `Package A imports X packages.`. This could be enhanced by adding an extra option for more detailed stats. For instance `Package A imports X packages (3rd Party: Y, Standard Library: Z)`

**Comment by ramya-rao-a** [comment]
This extension depends on a lot of [Go tools](https://github.com/Microsoft/vscode-go/wiki/Go-tools-that-the-Go-extension-depends-on) already to provide each of the editing, navigating, diagnostics and debugging features that is considered essential for developing in Golang. 

Some users are already wary on so many tools being installed when all they needed was some basic features. We are also seeing a gradual scope creep over the past year as this extension gets more popular and well meaning users have cool ideas such as yours to augment this extension.

Long story short, I do believe this is a cool feature to have. There are other related feature requests on how we can leverage godoc to provide more information right in VS Code instead of moving context to the browser. See #789. I would suggest to have a separate extension that takes a dependency on this extension and can have all the godoc related features.

**Comment by dastergon** [comment]
I totally understand, and it makes sensel. Thanks :+1
